RELEASING.md

```diff
@@ -1,23 +1,32 @@
 # Releasing
 
-Builds and releases are automated with GitHub Actions and GoReleaser. There are a few manual steps to complete:
+Builds and releases are automated with GitHub Actions and GoReleaser. There are several manual steps to complete:
 
-1. Start the release action: 
+1. Decide on the version number by reviewing commits since the last release. The provider versions follow [semantic versioning](https://semver.org/).
+
+2. Start the release action:
    
-  ```shell
-  git checkout master
-  git tag vX.Y.Z
-  git push upstream tag vX.Y.Z
-  ```
+   Assuming `upstream` is the remote name pointing to the canonical/terraform-provider-maas repository:
+   1. First, checkout the master branch and pull the latest changes:
 
-  Where `upstream` is the remote name pointing to the canonical/terraform-provider-maas repository. 
-  
-  The provider versions follow [semantic versioning](https://semver.org/), and the release action can be viewed under [Actions](https://github.com/canonical/terraform-provider-maas/actions).
+   ```shell
```

Comment:
this nested block requires at least two white spaces at the beginning of each line, including backtick lines